test/adapters/IDB_test.js

```diff
@@ -0,0 +1,214 @@
+"use strict";
+
+import chai, { expect } from "chai";
+import chaiAsPromised from "chai-as-promised";
+import sinon from "sinon";
+
+import IDB from "../../src/adapters/IDB.js";
+
+chai.use(chaiAsPromised);
+chai.should();
+chai.config.includeStack = true;
+
+const root = typeof window === "object" ? window : global;
+
+describe("adapters.IDB", () => {
```

Comment:
Is it possible to make it so it is possible to reuse this test to make sure another adapter works?

Something like: https://github.com/mozilla-services/loop-server/blob/master/test/storage_test.js#L32
And: https://github.com/mozilla-services/loop-server/blob/master/test/storage_test.js#L1090-L1092